website/_guides/arguments.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: 'guide'
+title: 'Arguments'
+menuOrder: 2
+redirect_from:
+  - /guides/usage
+---
+
+
+# Arguments
+TypeDoc accepts most of the command line arguments that the TypeScript compiler accepts. All
+arguments that are passed in without a flag will be parsed as input files. TypeDoc accepts
+directories as input files.
+
+To create documentation for an entire project via the CLI you can type:
+
+```bash
+$ typedoc --out path/to/documentation/ path/to/typescript/project/
+```
+
+### Help
+```bash
+$ typedoc --help
+```
+The help command will print the following listing of available commands.
+
+```bash
+{% include help_output.txt %}
+```
+
+### out
+
+```bash
+$ typedoc --out <path/to/documentation/>
+```
+
+Specifies the location the documentation should be written to.
+
+
+### name
+
+```bash
+$ typedoc --name <Documentation title>
+```
+
+Set the name of the project that will be used in the header of the template.
+
+
+### readme
+
+```bash
+$ typedoc --readme <path/to/readme|none>
+```
+
+Path to the readme file that should be displayed on the index page. Pass none to disable the index page and start the documentation on the globals page.
+
+
+### module
+
+```bash
+$ typedoc --module <commonjs or amd>
+```
+
+Specify module code generation: "commonjs" or "amd"
+
+
+### target
+
+```bash
+$ typedoc --target <ES3 or ES5>
+```
+
+Specify ECMAScript target version: "ES3" (default), or "ES5"
+
+
+### exclude
+
+```bash
+$ typedoc --exclude <pattern>
+```
+
+Exclude files by the given pattern when a path is provided as source
+
```

Comment:
Add a note on excluding multiple patterns, as many people wonder how. As of now, the best practice appears to be put an array of strings of patterns to exclude inside `typedoc.js` as specified by @dupski in [#170](https://github.com/TypeStrong/typedoc/issues/170#issuecomment-388607358).

```suggestion

Multiple files could also be excluded by adding an array of strings in `typedoc.js` like this in the `exclude` key:
```
module.exports = {
    out: './docs/dist/api/',
    readme: 'none',
    includes: './',
    exclude: [
        '**/__tests__/**/*',
        '**/__test_utils__/**/*',
        '**/__fixtures__/**/*',
        '**/testsuite/**/*'
    ],
    mode: 'file',
    excludeExternals: true,
    excludeNotExported: true,
    excludePrivate: true
};
```

```